core-services/prow/02_config/openshift/node-feature-discovery/_prowconfig.yaml

```
@@ -3,6 +3,7 @@ tide:
   - includedBranches:
     - release-4.0
     - release-4.10
+    - release-4.11
```

Given that 4.12 has already branched:
```suggestion
    - release-4.11
    - release-4.12
```